saos-persistence/src/main/java/pl/edu/icm/saos/persistence/model/JudgmentTextContent.java

```diff
@@ -0,0 +1,98 @@
+package pl.edu.icm.saos.persistence.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import org.apache.commons.lang3.StringUtils;
+
+import pl.edu.icm.saos.persistence.common.DataObject;
+
+/**
+ * Model class for judgment content
+ * 
+ * @author madryk
+ */
+@Entity
+@Table( uniqueConstraints = { @UniqueConstraint(columnNames = {"fk_judgment"})})
+@SequenceGenerator(name = "seq_judgment_text_content", allocationSize = 1, sequenceName = "seq_judgment_text_content")
+public class JudgmentTextContent extends DataObject {
+
+    private Judgment judgment;
+    private String rawTextContent;
+    private ContentType type;
+    private String path;
+    
+    /**
+     * Defines type of content stored under {@link JudgmentTextContent#getPath()} 
+     */
+    public enum ContentType {
+        PDF,
+        DOC
```

Comment:
It would be nice to have a short description/ comment of the values of ContentType. PDF is selfexplanatory but DOC?